doc/classes/CanvasItem.xml

```diff
@@ -648,7 +648,7 @@
 		</signal>
 		<signal name="hidden">
 			<description>
-				Emitted when the [CanvasItem] becomes hidden, either because its own [member visible] property was set to [code]false[/code] or the [member visible] property of one of its [CanvasItem] ancestors was set to [code]false[/code].
+				Emitted when the [CanvasItem] becomes hidden, the equivalent of when the return value of [method is_visible_in_tree] becomes [code]false[/code].
```

Comment:
Seems simpler to me, not sure if we like brackets though (I do :heart:). @Mickeon?
```suggestion
				Emitted when the [CanvasItem] becomes hidden (when the return value of [method is_visible_in_tree] becomes [code]false[/code]).
```

Anyway it's correct, it's only about the wording.

But as I've mentioned in https://github.com/godotengine/godot/pull/97367#discussion_r1771707628 the previous wording also seems fine/acceptable.

Reply:
I personally would like to learn into the existing description. Mentioning the returned value like this feels awkward. It's hard to explain right now. It makes it seem more complicated than it is, and it's a bit harder to understand when localized.
```suggestion
				Emitted when the [CanvasItem] becomes hidden, either because its own [member visible] property was set to [code]false[/code] or the [CanvasItem] is no longer visible in the tree (see [method is_visible_in_tree]).
```

Reply:
Because of the behavior @kleonc demonstrated here, I ended up merging your two suggestions, leaving out the 'either because its own [member visible] property was set to [code]false[/code]' part and fully committing to mentioning `is_visible_in_tree`:

`Emitted when the [CanvasItem] is no longer visible in the tree (see [method is_visible_in_tree]).`

Reply:
@Mickeon Your suggestion reads nice for sure. :+1: The question is whether it's clear. Imagine you have:

![Godot_v4 3-stable_win64_HZOP5yE0h2](https://github.com/user-attachments/assets/0f2ab123-a617-4b0b-9a5e-04ef25488eb8)

and you want to know whether setting `Child.visible = false` will result in emitting `Child.hidden` signal. Hence you read `hidden`'s docs. Is it clear to you from your suggested description whether it would be emitted?

(I wouldn't be sure and would need to test it out / check the source.)